app/src/pages/Home.tsx

```diff
@@ -16,7 +16,7 @@ const Home = () => {
   const { BACKEND_URL, FRONTEND_URL } = Constants;
   const { state: authState } = useAuthService();
   const isAdmin = authState.userInfo.client_roles?.includes('admin');
```

Comment:
Just a heads up. This may still throw an error if the user doesnt have any client roles. Test it, and if it errors, use the implementation here under the hasRole function: https://github.com/bcgov/keycloak-react/blob/main/src/service/useAuthService.js

Reply:
That was why I had to add the ?. Otherwise it assumed there would be a role. Now it just comes back as undefined if no roles exist, which works to say that they're not an admin.